package user.exception

object UserAlreadyExistsException {
}